app/services/manifest_builder.rb

```diff
@@ -611,8 +611,15 @@ def geotiff_child
           return false unless parent_node.is_a?(ScannedMapNode)
           wayfinder = Wayfinder.for(resource)
           parent_wayfinder = Wayfinder.for(wayfinder.parent)
-          if parent_wayfinder.mosaic_file_count == 1
-            parent_wayfinder.uncropped_geotiff_filesets.first
+          raster_resource = parent_wayfinder.raster_resources&.first
```

Comment:
Can I get some comments about what these individual use cases are? I'm having a tough time following.

Reply:
This took me a long time to untangle as well. The main issue is that in cases where a single scanned map (not part of a MapSet) has a raster child, there is only one geotiff fileset (versus two for the MapSet mosaic case). 

ScannedMap -> RasterResource -> GeoToff FileSet (target set to tiles - uncropped)

vs.

ScannedMap -> ScannedMap -> RasterResource -> GeoTiff FileSet (target set to tiles) + GeoTiff FileSet (no target - uncropped)

The old logic only returned uncropped GeoTiff files sets with the target set to 'tiled", which doesn't work for the single image non-mosaiced resources.

Reply:
Added comments to the method.